DependencyInjection/Compiler/DoctrineMongoDBMappingsPass.php

```diff
@@ -128,9 +128,7 @@ public static function createPhpMappingDriver(array $mappings, array $managerPar
      */
     public static function createAnnotationMappingDriver(array $namespaces, array $directories, array $managerParameters, $enabledParameter = false, array $aliasMap = array())
     {
-        $arguments = array(new Reference('doctrine_mongodb.odm.metadata.annotation_reader'), $directories);
-        $locator = new Definition('Doctrine\Common\Persistence\Mapping\Driver\SymfonyFileLocator', $arguments);
-        $driver = new Definition('Doctrine\ODM\MongoDB\Mapping\Driver\AnnotationDriver', array($locator));
+        $driver = new Definition('Doctrine\ODM\MongoDB\Mapping\Driver\AnnotationDriver', array(new Reference('doctrine_mongodb.odm.metadata.annotation_reader'), $directories));
```

Comment:
I suggest using ``annotation_driver`` rather than the old alias (which was kept only for BC reasons)